providers/shared/components/component.ftl

```diff
@@ -592,11 +592,18 @@
             "Type" : BOOLEAN_TYPE,
             "Default" : false
         },
+        {
+            "Names" : "EnableLogging",
+            "Description" : "WAF Logs in the OpsData DataBucket.",
+            "Type" : BOOLEAN_TYPE,
+            "Default" : true
+        },
```

Comment:
To keep it constistent with the changes above should this be 

```suggestion
        {
            "Names" : "Logging",
            "Children" : [ 
                {
                    "Names" : "Enabled",
                    "Type" : BOOLEAN_TYPE,
                    "Default" : true
                }
            ]
        },
```

Reply:
Updated in bc14a90

Reply:
@roleyfoley should we also update the `EnableLogging` attr on `apigateway` within the `cloudfront` attribute (i.e `solution.CloudFront.EnableLogging` to `solution.CloudFront.Logging.Enabled` and similarly the one on the CDN component?